### PR TITLE
Update introduction

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock113 <sherlockxu@yunify.com>

Update Introduction for v3.0

### CODE REVIEW

File: content/en/introduction/what-is-kubesphere.md [hunk 1]
```diff
@@ -6,38 +6,57 @@ description: 'What is KubeSphere'
...
-<iframe width="560" height="315" src="https://www.youtube.com/embed/u5lQvhi_Xlc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+- **Enhanced Security**. Security has alway remained one of our focuses in KubeSphere. In this connection, feature enhancements can be summarized as follows:
 
-## What is New in 2.1
+  - **Auditing**. Records will be kept to track who does what at what time. The support of auditing is extremely important especially for traditional industries such as finance and banking.
 
-We decouple some main feature components and make them pluggable and optional to choose so that users can install a default KubeSphere with resource requirements down to 2 cores CPU and 4G memory. Meanwhile, there are great enhancements in application store, especially in application lifecycle management.
+  - **Network Security and Isolation**. Security policies are created based on the network of tenants. KubeSphere 3.0 supports the management of security policies of tenant network, which can be completely isolated from each other. Users themselves can also decide whether services are accessible externally.
```
Comment (FeynmanZhou) [inline content/en/introduction/what-is-kubesphere.md:33]: Network Security => Network Policy.

Comment (FeynmanZhou) [inline content/en/introduction/what-is-kubesphere.md:33]: `Security policies are created based on the network of tenants. KubeSphere 3.0 supports the management of security policies of tenant network` is not exact, KubeSphere doesn't strengthen the network security, and there is no `security policy` in Kubernetes. Please take a look at these two pages to know what is NetworkPolicy.

- [Securing Kubernetes Cluster Networking
](https://ahmet.im/blog/kubernetes-network-policy/)
- [NetworkPolicy](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
-

File: content/en/introduction/what-is-kubesphere.md [hunk 2]
```diff
@@ -6,38 +6,57 @@ description: 'What is KubeSphere'
...
+  - **Network Security and Isolation**. Security policies are created based on the network of tenants. KubeSphere 3.0 supports the management of security policies of tenant network, which can be completely isolated from each other. Users themselves can also decide whether services are accessible externally. 
 
-It is worth mentioning that both DevOps and observability components have been improved significantly. For example, we add lots of new features including Binary-to-Image, dependency caching support in pipeline, branch switch support and Git logs output within DevOps component. We also bring upgrade, enhancements and bugfix in storage, authentication and security, as well as user experience improvements. See [Release Notes For 2.1.0](../../release/release-v210) for details.
+  - **Support Open Policy Agent**. Users can manage their security and authorization policies in a unified way with a general architecture.
```
Comment (FeynmanZhou) [inline content/en/introduction/what-is-kubesphere.md:35]: **Support Open Policy Agent** => **Provide flexible, fine-grained control based on Open Policy Agent**

File: content/en/introduction/what-is-kubesphere.md [hunk 3]
```diff
@@ -6,38 +6,57 @@ description: 'What is KubeSphere'
...
+As we adopt the open source model, development is proceeding in an open way and driven by KubeSphere community. KubeSphere is **100% open source** and available on [GitHub](https://github.com/kubesphere/) where you can find all the source code, documents and discussions. It has been widely installed and used in development, testing and production environments, and a large number of services are running smoothly in KubeSphere.
 
 ## Roadmap
 
 ### Express Edition -> KubeSphere 1.0.x -> KubeSphere 2.0.x -> KubeSphere 2.1.x -> KubeSphere 3.0.0
 
 ![Roadmap](https://pek3b.qingstor.com/kubesphere-docs/png/20190926000413.png)
 
-## Landscapes
+## Prospect and Aspiration
```
Comment (FeynmanZhou) [inline content/en/introduction/what-is-kubesphere.md:53]: Remove this paragraph please, I think the prospect is not proper to be included in the introduction.

File: content/en/introduction/what-is-kubesphere.md (hunk 1 above)
Comment (Sherlock113) [reply content/en/introduction/what-is-kubesphere.md:33]: As for the network security, I found some coverage and I would like to make sure if they are correct:
http://www.cnbp.net/news/detail/24087
截屏2020-08-08下午1 49 28
http://m.ce.cn/bwzg/202006/30/t20200630_35227640.shtml
截屏2020-08-08下午1 49 51

Comment (FeynmanZhou) [reply content/en/introduction/what-is-kubesphere.md:33]: `NetworkPolicy` is much professional and common-used in Kubernetes, users are easy to understand the capabilities. You can ignore those pieces of Chinese news reports.

**Pull request merged**